mysite/code/controllers/SupportedAddonsApiController.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+class SupportedAddonsApiController extends Controller
+{
+    private static $allowed_actions = [
+        'index',
+    ];
+
+    /**
+     * Set the default cache lifetime in seconds. Only used outside of "dev" environments.
+     *
+     * @config
+     * @var int
+     */
+    private static $cache_age = 86400; // 24 hours
```

Comment:
Can you put the "24 hours" part into the PHPdoc above over an inline comment?

Also we could probably cache this even more aggressively if you wanted to. The list of supported modules won't change very often at all

Reply:
👍 I set the age to a week.